pkg/nsx/services/mediator/mediator.go

```diff
@@ -14,3 +23,25 @@ type ServiceMediator struct {
 	*securitypolicy.SecurityPolicyService
 	*vpc.VPCService
 }
+
+// GetOrgProjectVPC is a common method, extracting the org and project string from vpc path of the VPC model.
+// VPC path looks like "/orgs/default/projects/project-1/vpcs/vpc-1",
```

Comment:
From the comment, the input should be a string. I didn't get why you use the ns as the input, I think it's not common enough.

Reply:
The comment explains how to extract, and the input is indeed a namespace.